delayed_job.rb

```diff
@@ -1,12 +1,20 @@
-dep 'delayed job', :env, :user do
-  requires 'delayed_job.upstart'.with(env, user)
+dep 'delayed job', :env, :user, :queue do
+  requires 'delayed_job.upstart'.with(env, user, queue)
 end
 
-dep 'delayed_job.upstart', :env, :user do
+dep 'delayed_job.upstart', :env, :user, :queue do
   respawn 'yes'
+
+  queue.default 'global'
```

Comment:
yeah, so I was checking for it so that it would keep the exisiting global filename as is which is

```
theconversation.com_delayed_job.conf
```